templates/download/server/manual.html

```diff
@@ -523,15 +669,16 @@ <h3 class="p-heading--5">
       const targetTab = document.querySelector(".p-tabs__link[href='#manual-install']");
 
       targetTab.addEventListener("click", () => {
-        const releaseNotesTabContainer = document.getElementById("release-notes");
-        releaseNotesTabContainer.classList.remove("u-hide")
-      })
+        handleTab("first-child-tab", "release-notes-tab-lts", "release-notes-lts");
+        handleTab("second-child-tab", "release-notes-tab-latest", "release-notes-latest");
+      });
     }
-    showBaseNestedTab()
+
 
     addEventListener("DOMContentLoaded", () => {
-      handleNonTabHashClick()
-      showParentTab()
+      handleNonTabHashClick();
+      showParentTab();
+      showBaseNestedTab();
```

Comment:
@minkyngkm Can you please explain what does moving this function call to an event change?

Reply:
I don't think it affects how the tab works much, but it's working fine after this change. `showBaseNestedTab` is used when the manual install tab is clicked, which means the user can access the element. so, it doesn't have to be inside the addEventListener i think, but there’s no harm in placing it there with the other functions so that it runs when the DOM is ready